# LibrarySystem
### please change  '[config.py](./app/config.py)' in line 5 to connect your datebase.
